src/importexport/ove/internal/ove.cpp

```diff
@@ -4405,7 +4405,7 @@ bool OvscParse::parse()
     Block placeHolder;
 
     m_handle = &handle;
-    auto _ = ResetToNull(&m_handle);
+    auto _ = ResetToNull<StreamHandle>(&m_handle);
```

Comment:
I like my `<typeof(*handle_)>` better

Reply:
Would it be better to add a free function so the template-deduction happens regardless of version?

Reply:
If possible without jumping through burning hoops ;-)

Reply:
But for master it might not be needed, your orinal version is just fine, as we're having C++17 there as a fixed requirement.

Reply:
The hoops were lukewarm at best, so I added it—I think it looks a lot nicer.